lookout/style/format/features.py

```diff
@@ -163,6 +176,12 @@ def __init__(self, language: str, siblings_window: int = 5, parents_depth: int =
         language = language.lower()
         self.tokens = importlib.import_module("lookout.style.format.langs.%s.tokens" % language)
         self.roles = importlib.import_module("lookout.style.format.langs.%s.roles" % language)
+        try:
+            self.token_unwrappers = importlib.import_module(
+                "lookout.style.format.langs.%s.token_unwrappers" % language).TOKEN_UNWRAPPERS
+        except ImportError:
+            # It's normal for some languages not to have a token_unwrappers module.
+            pass
```

Comment:
We should init `self.token_unwrappers` to empty dict here otherwise we hit AttributeError